Color.py

```diff
@@ -0,0 +1,92 @@
+# this class is solely for the purpose of
+# making makeLighter makeDarker work.
+# both of these functions destructively modify a color
+# and a color in java is a constant value so we have to put
+# this python interface here
+#
+# Buck Scharfnorth (28 May 2008): Modified to no longer assume the value is 0-255
+# and the gray Color constructor to allow only 1 color parameter (will
+# take 2, but ignores the second)
+
+# JRS -- 2020-06-23 -- TEMPORARY FUNCTION UNTIL PIXEL CLASS IS AVAILABLE
+def PixelcorrectLevel(c):
+    """Return pixel value in range [0..255]
+    """
+    return c % 256
+
+import math
+from tkinter import colorchooser
+class Color:
+
+    def __init__(self, r, g=None, b=None):
+        if b == None:
+            if isinstance(r, Color):
+                self.color = r
+            else:
+                val = PixelcorrectLevel(r)
```

Comment:
Because having Color import pixel would cause circular inheritance I combined the Color and Pixel class into the pixel class. Thus when wanting to use color one would call from Pixel import Color